.test-defs/pod-e2e.yaml

```diff
@@ -0,0 +1,17 @@
+kind: TestDefinition
+metadata:
+  name: pod-e2e-test
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Terraformer Pod E2E test
+  activeDeadlineSeconds: 1800
```

Comment:
The teststep timeout should be little bit higher that the timeout that is defined in the go test.

Reply:
Yep, done. Also, decreased the overall timeouts.